### ISSUE TITLE
ScrollArea blocks key input on contenteditable div (Slate.js, Draft.js)

### ISSUE THREAD

**Issue by santiagopuentep** [souhe/reactScrollbar#77]
ScrollArea blocks arrow keys movement of cursor inside contenteditable `div` and instead scrolls the scroll area. Slate.js and Draft.js editors inside `react-scrollbar` get blocked by this.

**Comment by rowdyroad** [comment]
+1

**Comment by galanovnick** [comment]
+1

**Comment by oleg-cnnr** [comment]
+1 for textarea element [here](https://github.com/souhe/reactScrollbar/blob/master/src/js/ScrollArea.jsx#L277)

**Comment by olup** [comment]
+1 on textArea and Draft.js

**Comment by APiercey** [comment]
In my app, I'm not using using any key events for react-scrollbar. But this is an issue for my Draft.js components. So I monkey-patched it out for my app.

This is what I did:
```
import ScrollArea from 'react-scrollbar'
ScrollArea.prototype.handleKeyDown = Function
```

**Comment by souhe** [comment]
Fixed in version 0.5.2 Thanks to @mikhailmoshnogorsky :)
